src/index.html

```diff
@@ -1,27 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-B4NRS6N0MV"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-NYJJGE8VVD"
+    ></script>
     <script>
       window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
+      function gtag() {
+        dataLayer.push(arguments);
+      }
       gtag('js', new Date());
 
-      gtag('config', 'G-B4NRS6N0MV');
+      gtag('config', 'G-NYJJGE8VVD');
     </script>
-    
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
     <title>Document</title>

```

Comment:
- [ ] **Required:** This title is not a valid title. Kindly change it such that it describes its content. Example Mwafrika's Portfolio, Mwafrika Mufungizi Josué, Mwafrika Mufungizi's Portfolio, etc.